cmd/osde2e/cleanup/cmd.go

```diff
@@ -225,6 +234,16 @@ func run(cmd *cobra.Command, argv []string) error {
 		}
 	}
 
+	if args.ec2 {
+		ec2InstanceTerminatedCounter := 0
+		ec2InstanceFailedCounter := 0
+		err = aws.CcsAwsSession.TerminateEC2Instances(fmtDuration, args.dryRun, args.sendSummary, &ec2InstanceTerminatedCounter, &ec2InstanceFailedCounter, &ec2ErrorBuilder)
```

Comment:
I know everything else does it, so we can go with it but it is a bit odd to pass pointers to ints as parameters rather than just returning the counters as return values from `TerminateEC2Instances`. Just FWIW

Reply:
I didn't want to change the function's return type. As it would have to return an error along with counter. Please do let me know if that way is more intuitive.